scalatest/src/main/scala/org/scalatest/AsyncEngine.scala

```diff
@@ -850,7 +850,7 @@ private[scalatest] sealed abstract class AsyncSuperEngine[T](concurrentBundleMod
   private[scalatest] def testTags(testName: String, theSuite: Suite): Set[String] = {
     // SKIP-SCALATESTJS,NATIVE-START
     val suiteTags = for { 
-      a <- theSuite.getClass.getAnnotations
+      a <- theSuite.getClass.getAnnotations.map(_.nn)
```

Comment:
**Java method returns array**
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.